src/main/java/com/dev/cinema/util/HibernateUtil.java

```diff
@@ -0,0 +1,24 @@
+package com.dev.cinema.util;
+
+import com.dev.cinema.exceptions.DataProcessingException;
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+public class HibernateUtil {
+    private static SessionFactory sessionFactory = initSessionFactory();
+
+    private HibernateUtil(){
+    }
+
+    public static SessionFactory getSessionFactory() {
+        return sessionFactory;
+    }
+
+    private static SessionFactory initSessionFactory() {
+        try {
+            return new Configuration().configure().buildSessionFactory();
+        } catch (Exception e) {
+            throw new DataProcessingException("Error creating Session Factory", e);
```

Comment:
its a different kind of exception, let's leave just RuntimeException here

Reply:
fixed